models/chat/telegram.py

```diff
@@ -1,13 +1,13 @@
 from re import compile as re_compile
 from requests import get, ConnectionError, exceptions, Timeout
-from logging import getLogger
+
+
 
 class Telegram():
     def __init__(self, token='', client_id=''):
         self.api = 'https://api.telegram.org/bot'
         self._token = token
         self._client_id = str(client_id)
-        self.logger = getLogger('pyCryptoBot')
```

Comment:
@emreeroglu, how or why is this different to other classes? In some you just call "logging" directly after importing but in this one you use getLogger? Is this a mistake or a specific reason to do this?

Reply:
Actually we can call logging.getLogger('LoggerName') anywhere we want.
It would return same logger configurations set once at the creation of that logger.
We can user LoggerName to distinguish logs from each other, this is a good feature.

In our case I just tried to get all logging in one place so we can start with something basic and working.
After this we can see where do we want to go.